monai/networks/blocks/selfattention.py

```diff
@@ -38,11 +38,7 @@ def __init__(
         dim_head: int | None = None,
         qkv_bias: bool = False,
         save_attn: bool = False,
-        causal: bool = False,
-        sequence_length: int | None = None,
-        rel_pos_embedding: Optional[str] = None,
-        input_size: Optional[Tuple] = None,
-        attention_dtype: Optional[torch.dtype] = None,
```

Comment:
Why have these parameters been removed? They are used below in the init, and they are initialised in the SABlock definition in some pieces of code, like the Transformers.